WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -147,6 +159,7 @@ private void StepOutputRegistrationPhase()
 					{
 						var diffMoney = Money.Satoshis(diff);
 						coinjoin.Outputs.AddWithOptimize(diffMoney, Config.BlameScript);
+						round.LogInfo("Filled up the outputs to build a reasonable transaction because some alice failed to provide its output.");
```

Comment:
Are you allowed to log `diffMoney` or not (to stay zero knowledge company)? 

If you are, then `diffMoney` is probably a good candidate to log.

Reply:
I don't think so. @nopara73

Reply:
@nopara73 This is the most important question here.

Reply:
This is just for blame round, the tx will be thrown away because the bad guy won't sign it. We don't need the details here.